docs/bucket/notifications/README.md

```diff
@@ -979,22 +979,24 @@ MINIO_NOTIFY_MYSQL_COMMENT      (sentence)           optionally add a comment to
 
 `dsn_string` is optional, if not specified, the connection information specified by the `host`, `port`, `user`, `password` and `database` parameters are used.
 
-MinIO supports persistent event store. The persistent store will backup events when the MySQL connection goes offline and replays it when the broker comes back online. The event store can be configured by setting the directory path in `queue_dir` field and the maximum limit of events in the queue_dir in `queue_limit` field. For eg, the `queue_dir` can be `/home/events` and `queue_limit` can be `1000`. By default, the `queue_limit` is set to 10000.
+MinIO supports persistent event store. The persistent store will backup events when MySQL connection goes offline and replays it when the broker comes back online. The event store can be configured by setting the directory path in `queue_dir` field and the maximum limit of events in the queue_dir in `queue_limit` field. For eg, the `queue_dir` can be `/home/events` and `queue_limit` can be `1000`. By default, the `queue_limit` is set to 10000.
```

Comment:
```suggestion
MinIO supports persistent event store. The persistent store will backup events if the MySQL connection goes offline and replays them when the broker comes back online. The event store can be configured by setting the directory path in `queue_dir` field and the maximum limit of events in the queue_dir in `queue_limit` field. For eg, the `queue_dir` can be `/home/events` and `queue_limit` can be `1000`. By default, the `queue_limit` is set to 10000.
```

Reply:
I changed the sentence like below in the light of your comment:
```
MinIO supports persistent event store. The persistent store will backup events if MySQL connection goes offline and replay the events when the broken connection comes back online. ```